smarts/sstudio/tests/test_generate.py

```diff
@@ -130,7 +130,7 @@ def fake_map_builder(map_spec: MapSpec) -> bool:
 
 
 def test_generate_od_map():
-    scenario_root = "scenarios/od_4lane"
+    scenario_root = "scenarios/smarts/od_4lane"
```

Comment:
Is it `smarts` or `open_drive`?
```suggestion
    scenario_root = "scenarios/open_drive/od_4lane"
```